include/drivers/sensor.h

```diff
@@ -81,6 +81,15 @@ enum sensor_channel {
 	SENSOR_CHAN_AMBIENT_TEMP,
 	/** Pressure in kilopascal. */
 	SENSOR_CHAN_PRESS,
+	/** Euler heading(yaw) in degrees */
+	SENSOR_CHAN_EULER_H,
+	/** Euler roll in degrees */
+	SENSOR_CHAN_EULER_R,
+	/** Euler pitch in degrees */
+	SENSOR_CHAN_EULER_P,
+	/** Euler heading, roll and pitch in degrees */
+	SENSOR_CHAN_EULER_HRP,
```

Comment:
I'd change the definitions to:

```suggestion
	/** Yaw/Heading angle in degrees */
	SENSOR_CHAN_YAW,
	/** Pitch angle in degrees */
	SENSOR_CHAN_PITCH,
	/** Roll angle in degrees */
	SENSOR_CHAN_ROLL,
	/** Yaw, pitch and roll andles in degrees */
	SENSOR_CHAN_YPR,
```

As far as I know, yaw, pitch and roll are always given as Euler angles.